metadata.json

```diff
@@ -1,6 +1,6 @@
 {
-    "title": "CLDF dataset of the supplementary materials to "Subgrouping in a ‘dialect continuum’: A Bayesian phylogenetic analysis of the Mixtecan language family" (Auderset et al. 2023. JoLE)",
+    "title": "CLDF dataset accompanying Auderset et al.'s \"Subgrouping in a ‘dialect continuum’\" from 2023",
     "url": "https://github.com/lexibank/mixtecansubgrouping",
     "license": "CC-By-4.0",
-    "citation": "Auderset, Sandra, Simon J. Greehnhill, Christian T. DiCanio, Eric W. Campbell. 2023. CLDF data set of Supplementary Materials to "Subgrouping in a `dialect continuum': A Bayesian phylogenetic analysis of the Mixtecan language family". Journal of Language Evolution, tbd. https://doi.org/10.5281/zenodo.6513506"
+    "citation": "Auderset, Sandra, Simon J. Greehnhill, Christian T. DiCanio, Eric W. Campbell. (2023) \"Subgrouping in a `dialect continuum': A Bayesian phylogenetic analysis of the Mixtecan language family\". Journal of Language Evolution, tbd"
```

Comment:
Just leave out the tbd and end by a full stop, as all citations should end, then it is fine.

Reply:
Okay, thank you! I adjusted it accordingly.